### PR TITLE
Pr/safari performance boost

### PR DESCRIPTION
**Pull request by izzappel**
- [ ] Select-Dropdown does not scroll
- [ ] Select-Dropdown behind Modal

### CODE REVIEW

File: packages/tocco-ui/src/Select/DropdownIndicator.js [hunk 1]
```diff
@@ -2,14 +2,26 @@ import React from 'react'
...
-import Ball from '../Ball'
+import {StyledDropdownIndicatorBall, StyledDropdownIndicatorWrapper} from './StyledComponents'
 
 const DropdownIndicator = props => {
-  return !props.selectProps.isDisabled && <components.DropdownIndicator {...props}>
-    <Ball
-      icon={props.selectProps.menuIsOpen ? 'chevron-up' : 'chevron-down'}
-      tabIndex={-1}
-    />
+  if (props.selectProps.isDisabled) {
```
Comment (yilkur) [inline packages/tocco-ui/src/Select/DropdownIndicator.js:8]: destructure props?

File: packages/tocco-ui/src/Select/DropdownIndicator.js [hunk 2]
```diff
@@ -2,14 +2,26 @@ import React from 'react'
...
+    return null
+  }
+
+  return <components.DropdownIndicator {...props}>
```
Comment (yilkur) [inline packages/tocco-ui/src/Select/DropdownIndicator.js:12]: use () for cleaner structure

File: packages/tocco-ui/src/Select/SingleValue.js [hunk 3]
```diff
@@ -1,13 +1,45 @@
...
+ */
+const ignoredAttributes = ['linkComp']
+const areEqual = (prevProps, nextProps) => {
+  const diff = js.difference(prevProps, nextProps)
+  const clean = _omitBy(_omit(diff, ignoredAttributes), v => Object.keys(v).length === 0)
+  return Object.entries(clean).length === 0
+}
+
+const Content = memo(({linkComp: DetailLink, entityKey, children}) => {
+  return (DetailLink
```
Comment (yilkur) [inline packages/tocco-ui/src/Select/SingleValue.js:27]: return statement can be omitted with () I think? ^^

**Review by stewue — APPROVED** [review]

**Review by yilkur — APPROVED** [review]

**Comment by izzappel** [comment]
No changes to the select dropdown for now.
New PR created: https://github.com/tocco/tocco-client/pull/2086

**Pull request closed without merging**